setup.py

```diff
@@ -1,53 +1,42 @@
 from setuptools import setup, find_packages
-
 import os
 
+with open(os.path.join(os.path.dirname(__file__), "requirements.txt"), "r") as file:
+    lines = [ln.strip() for ln in file.readlines()]
+
+packages_pip = []
 
-PATH_ROOT = os.path.dirname(__file__)
+for ln in lines:
+    if ln.startswith("#"):
+        continue
+    if ln:  # if requirement is not empty
+        packages_pip.append(ln)
 
-def load_requirements(path_dir=PATH_ROOT, file_name='requirements.txt', comment_char='#'):
-    """ Taken from the pl_bolts repo.
-    """ 
-    with open(os.path.join(path_dir, file_name), 'r') as file:
-        lines = [ln.strip() for ln in file.readlines()]
-    reqs = []
-    for ln in lines:
-        if ln.startswith(comment_char):
-            continue
-        if ln:  # if requirement is not empty
-            reqs.append(ln)
-    return reqs
 
 setup(
-    name='sequoia',
-    __version__ = "0.0.1",
+    name="sequoia",
+    version="0.0.2",
```

Comment:
No need to bump the version. This isn't even on pypi yet, but still.

Reply:
feel free to unbump it. But the version string should be `version`, not `__version__`